src/components/ItemView.vue

```diff
@@ -23,7 +24,7 @@ xr.open_dataset("${ props?.item?.assets?.data?.href }", engine="zarr")`)
             <div class="time" v-if="props.item?.properties?.datetime">{{ props.item.properties.datetime }}</div>
             <div class="time" v-if="props.item?.properties?.start_datetime && props.item?.properties?.end_datetime">{{ props.item.properties.start_datetime }} - {{ props.item.properties.end_datetime }}</div>
             <div class="keywords" v-if="props.item?.properties?.keywords"><ul><li v-for="kw in props.item.properties.keywords">{{ kw }}</li></ul></div>
-            <div class="license" v-if="props.item?.properties?.license">⚖️ {{ props.item.properties.license }}</div>
+            <div><License :item="props.item" /></div>
```

Comment:
I'd recommend pass only the license, this would make the component reusable also for cases where we don't want to show a stac item, e.g.:
```suggestion
            <div><License :spdx="props.item?.properties?.license" /></div>
```